pkg/model/crd.go

```diff
@@ -387,6 +387,82 @@ func (r *CRD) GetOutputWrapperFieldPath(
 	return &opConfig.OutputWrapperFieldPath
 }
 
+// GetOutputShape returns the Output shape for given operation.
+func (r *CRD) GetOutputShape(
+	// The operation to look for the Output shape
+	op *awssdkmodel.Operation,
+) *awssdkmodel.Shape {
+	if op == nil {
+		return nil
+	}
+
+	outputShape := op.OutputRef.Shape
+	if outputShape == nil {
+		return nil
+	}
+
+	// We might be in a "wrapper" shape. Unwrap it to find the real object
+	// representation for the CRD's createOp/DescribeOP.
+
+	// Use the wrapper field path if it's given in the ack-generate config file.
+	wrapperFieldPath := r.GetOutputWrapperFieldPath(op)
+	if wrapperFieldPath != nil {
+		wrapperOutputShape, err := r.GetWrapperOutputShape(outputShape, *wrapperFieldPath)
+		if err != nil {
+			msg := fmt.Sprintf("Unable to unwrap the output shape: %v", err)
+			panic(msg)
+		}
```

Comment:
Can we wrap and return an error instead? All the panic calls are made inside code generators functions - IMO CRD methods should just send simple errors

Reply:
Updated the code. Code now returns `nil`.

Reply:
Why does it return `nil` @kumargauravsharma ? Unless i'm missing something i think it should just return a wrapped error like: `fmt.Errorf("unable to unwrap the output shape: %v", err)`

Reply:
Ohhh i'm missing the fact that this function doesn't return an error.. I think that it would be nice to return an error anyways for debuging purposes

Reply:
Changed method return type to include error as well.